docassemble/ALKilnInThePlayground/data/questions/run_alkiln_tests.yml

```
@@ -1732,39 +1904,49 @@ need:
   - folder_path
   - get_zip_path
 code: |
-  # Let this block run regardless just in case it can manage to clean
-  # something up
+  # Note: Ensure this block run regardless of the outcome just in case it
```

```suggestion
  # TODO: Ensure this block runs regardless of the outcome just in case it
```

There are places where we don't accomplish this because there are errors we raise or don't catch.